package.json

```diff
@@ -27,7 +27,7 @@
         }
     ],
     "engines": {
-        "vscode": "^1.6.0"
+        "vscode": "^1.12.0"
```

Comment:
`v3.3.0` of the `vscode-languageclient` lib requires as least v`1.12.0` of the VSCode d.ts file, so I had to update this as well. That said, `1.12.0` is still a very old version of VS Code, so updating this shouldn't have any impact on backwards compatibility.